client/components/tinymce/plugins/simple-payments/style.scss

```diff
@@ -48,30 +48,59 @@
 		flex: none;
 		margin-bottom: 0;
 	}
-}
 
-.editor-simple-payments-modal__nudge-wrapper {
-	margin: 24px;
-}
+	.empty-content {
+		position: absolute;
+			top: 50%;
+			left: 50%;
+		transform: translate( -50%, -50% );
+		padding-top: 0;
+		padding-bottom: 73px;
+		margin: 0;
+		width: 80%;
+		overflow: visible;
+	}
 
-.editor-simple-payments-modal__nudge-title {
-	text-align: center;
-	margin-top: 100px;
-	font-size: 40px;
-}
+	.empty-content__illustration {
+		margin-bottom: 20px;
+	
+		@media ( max-height: 470px ) {
```

Comment:
There are some CSS guidelines regarding using cusstom `@media` queries: https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md#media-queries. Looks like we should try to use the `breakpoint` mixin as much as possible and ideally not define our own `media` queries.